apps/resources-wcm/src/main/webapp/skin/less/ecms/portlets/attachments/attachmentsApp.less

```diff
@@ -20,6 +20,11 @@
     i {
       font-size: 41px;
     }
+    &.smallAttachmentIcon {
+      i {
+        font-size: 24px;
+      }
+    }
```

Comment:
```suggestion
    &.smallAttachmentIcon i {
      font-size: 24px;
    }
```

Reply:
it won't work because the `tag i` already has `font-size` [here](https://github.com/exoplatform/ecms/pull/1255/files#diff-cb0bf7f36f5e7277f9351ead8e269b7ec4a8d8d9949831c558b9f04a5507fa78L21)

Reply:
should give exactly the same resulted CSS, no diff about both notations, just more few accolades and Less file lines